src/xcode/ENA/ENA/Source/Services/TicketValidation/TicketValidation.swift

```diff
@@ -43,5 +44,113 @@ final class TicketValidation: TicketValidating {
 	func cancel() {
 
 	}
-
+	
+	func encryptAndSignDCC(
```

Comment:
I think this does not necessarily have be part of the public API of `TicketValidation`, maybe it makes sense to move it to a separate struct or class and just use it from the `validate()` func of the `TicketValidation`? Not sure if we can already use it, maybe we need a new subtask to integrate the steps in here once we have all the parts available.

Reply:
Moved the code to a own component. Created task for integration.